### PR TITLE
Removing the environment string from the code and using os/user

### PR DESCRIPTION
**Pull request by uriel-fanelli**
*User.Username contains the string the code seems to want.

### CODE REVIEW

File: routers/install.go [hunk 1]
```diff
@@ -100,12 +101,14 @@ func Install(ctx *middleware.Context) {
...
+
+		// user.Current returns back a *User, while form.RunUser is a string
+		WhoAmI, err := user.Current()
+		if err == nil {
+			form.RunUser = WhoAmI.Username
+		} else {
+			form.RunUser = setting.RunUser
 		}
-	} else {
-		form.RunUser = setting.RunUser
```
Comment (lunny) [inline routers/install.go]: Why remove these two lines?

**Comment by uriel-fanelli** [comment]
This piece of code is setting the current username (if possible) or the one configured in git. We only need one "if" statement here because we don't have to choose between two env string, so at least one "else" is useless now.

**Comment by tboerger** [comment]
I think this one can get closed, right?

**Comment by lunny** [comment]
I think we can close this pr because I have accepted #20 @uriel-fanelli .

**Pull request closed without merging**